src/test/java/org/codehaus/mojo/versions/SetMojoTest.java

```diff
@@ -4,16 +4,14 @@
 import org.apache.maven.plugin.MojoExecutionException;
 import org.apache.maven.plugin.MojoFailureException;
 import org.apache.maven.project.MavenProject;
-import org.junit.Test;
+import org.codehaus.mojo.versions.utils.BaseMojoTestCase;
 
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.containsString;
 import static org.hamcrest.Matchers.is;
-import static org.junit.Assert.fail;
 
-public class SetMojoTest
+public class SetMojoTest extends BaseMojoTestCase
 {
-    @Test
```

Comment:
I would not remove `@Test` annotations ... yes I know that TestCase is from junit 3 ...

Reply:
Ok, restored.